includes/blocks.inc

```diff
@@ -108,3 +108,34 @@ function islandora_compound_object_jail_display_block() {
   }
   return $block;
 }
+
+/**
+ * Constructs the default navigation block for a compound.
+ *
+ * @return array
+ *   Renderable array for the block.
+ */
+function islandora_compound_object_navigation_block() {
```

Comment:
Moving functions around like this is an API change and has to go through our usual process for such. There are options 1. Deprecation cycle 2. This is minimal risk so with commiters buy in we can forgo deprecation and just move it and put the move in the release notes. 3. Don't do it.